scripts/bygg

```diff
@@ -213,9 +214,65 @@ build_container()
 DockerFileHere
 }
 
+get_short_machine_name() 
+{
+  [[ "$MACHINE" =~ ^mx4- ]] || { >&2 echo "MACHINE=$MACHINE does not start with mx4- as expected" ; return 1 ; }
+  [[ "$MACHINE" == "mx4-c61-rio" ]] && echo c61 || echo "${MACHINE:4}"
+}
+
 create_hm_update_image()
 {
-  echo "🟡 TODO MX-4 update image $image for $MACHINE"
+  echo "☕ Creating update image $image for $MACHINE"
+  short_machine="$(get_short_machine_name)"
+  cd "$YOCTO_FOLDER"
+
+  # Allow hostmobility/buildplatform-mx4:2.0, which runs as uid = 1000 to write
+  # to mx4-deploy. This directory was created by our crops/poky based container
+  # which runs as the current user which might be different.
+  chmod o+w mx4-deploy
+
+  cat >build_mx4-v2.sh <<'EOF'
+#!/bin/bash -e
+echo "Start build make_system.sh deploy only!"
+cd mx4-deploy
+
+# Clean up old directories. make_system.sh requires it!
+sudo rm -rf deploy-*
+
+DEPLOY_DIR=../build/tmp/deploy/images/$MACHINE
+
+if [[ "$SHORT_MACHINE" == "c61" ]]; then 
+  root_filesystem=$DEPLOY_DIR/${TARGET_IMAGE}-${MACHINE}.tar.xz
+else 
+  root_filesystem=$DEPLOY_DIR/${TARGET_IMAGE}-${MACHINE}.ext3
+fi
+
+dtb="$DEPLOY_DIR/tegra30-mx4-t30.dtb"
```

Comment:
Static dtb for t30 will not work for c61. see my old deploy script how to handle it.